src/js/redux/actions/googleAnalytics/googleAnalytics.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * googleAnalyticsActions.js
+ * Created by Jonathan Hill 07/14/21
+ */
+
+export const setIsInitialApplicationLoadForDAPGoogleAnalyticsToFalse = () => ({
```

Comment:
Could we shorten these names to something like `setInitialAppLoadForDAP` that gets the point across with fewer characters? We could use comments to provide more specifics